msal/src/main/java/com/microsoft/identity/nativeauth/statemachine/errors/Error.kt

```diff
@@ -105,7 +124,7 @@ class SubmitCodeError(
     override val errorCodes: List<Int>? = null,
     val subError: String? = null,
     override var exception: Exception? = null
-): SignInSubmitCodeResult, SignUpSubmitCodeResult, ResetPasswordSubmitCodeResult, Error(errorType = errorType, error = error, errorMessage= errorMessage, correlationId = correlationId, errorCodes = errorCodes, exception = exception)
+): SignInSubmitCodeResult, SignUpSubmitCodeResult, ResetPasswordSubmitCodeResult, BrowserRequiredError(errorType = errorType, error = error, errorMessage= errorMessage, correlationId = correlationId, errorCodes = errorCodes, exception = exception)
```

Comment:
Why SubmitCodeError use BrowserRequiredError here instead of Error but ResendCodeError use Error?

Reply:
For those valid INativeAuthCommandResult.Redirect Error, it should extends BrowserRequiredError